feelslikemonday/app/src/main/java/com/example/feelslikemonday/service/SortObjectDateTime.java

```diff
@@ -0,0 +1,17 @@
+package com.example.feelslikemonday.service;
+
+import com.example.feelslikemonday.model.followeeMoodEvent;
+import java.util.Comparator;
+
+/**
+ * This class contains a method that will compare mood events within a mood history
+ * It will sort them in reverse chronological order (based on the date and time the mood was posted)
+ */
+public class SortObjectDateTime implements Comparator<followeeMoodEvent>{
+
```

Comment:
We should not need this if we use mood histories from the user object

Reply:
once you get the most recent mood event from each followee's mood history, you need to sort them in reverse chronological order. So that's why this is here. I don't understand what you are proposing tbh.

Reply:
If we append events to the list in chronological order, we either need to get the first element or the last element, depending on how the events are ordered. I don't think we need this class since we only need to grab one event, but if you want to keep it we can

Reply:
basically since the lists are sorted in chronological order, you only need to get the item at index 0 of each user to get the most recent event. However, once you have all of the "most recent mood events" you cannot gaurentee that these most recent events are sorted. So thats the main reason behind this class

Reply:
so I vote we keep it

Reply:
Sounds good. I'm not entirely certain how the most recent mood events work so I trust your judgment.